src/index.html

```diff
@@ -5,8 +5,293 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <span class="header__logo logo">
+        <a href=""><img src="./images/logo.png" alt="logo"></a>
+      </span>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover">Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tabs" class="nav__link">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog-page">
+      <div class="catalog-page__cards cards">
+        <div class="cards__card product"
+             data-qa="card">
```

Comment:
коли є два атрибута, можна їх зилишити в одному рядку, або перенести всі атрибути, тоді також закриваючий тег слід перенести
```suggestion
        <div class="cards__card product" data-qa="card">
```
or
```suggestion
        <div
          class="cards__card product"
          data-qa="card"
        >
```